doc/languages-frameworks/python.section.md

```diff
@@ -997,13 +997,18 @@ and in this case the `python3` interpreter is automatically used.
 
 ### Interpreters {#interpreters}
 
-Versions 2.7, 3.8, 3.9, 3.10 and 3.11 of the CPython interpreter are available
-as respectively `python27`, `python38`, `python39`, `python310` and `python311`.
-The aliases `python2` and `python3` correspond to respectively `python27` and
-`python310`. The attribute `python` maps to `python2`. The PyPy interpreters
-compatible with Python 2.7 and 3 are available as `pypy27` and `pypy3`, with
-aliases `pypy2` mapping to `pypy27` and `pypy` mapping to `pypy2`. The Nix
-expressions for the interpreters can be found in
+| Packages   | Aliases         | Interpreters |
```

Comment:
```suggestion
| Package    | Aliases         | Interpreter |
```

One package, has one interpreter, but potentially multiple aliases.

Reply:
Fair point. Plurals can be misleading in table headers, and here I also see now that it's better not to use them to refer to the columns but per-row.

Header updated in latest commit.